openshift-ci/Dockerfile.tools

```diff
@@ -28,7 +28,7 @@ RUN mkdir ~/bin && \
     eval "$(gimme $GOVERSION)" && \
     cat $GIMME_ENV >> $HOME/.bashrc && \
     # get required golang tools and OC client
-    go install github.com/onsi/ginkgo/v2/ginkgo@v2.8.4 && \
+    go install github.com/onsi/ginkgo/v2/ginkgo && \
```

Comment:
same

Reply:
this needs a second look

Reply:
it was before the ginkgo/v2 changes
`go install github.com/onsi/ginkgo/ginkgo@latest && \`

Reply:
so maybe we do want here to have :
`go install github.com/onsi/ginkgo/v2/ginkgo@v2.8.4` ?
or even the latest?

Reply:
reverted back